src/kv/MemDB.h

```diff
@@ -147,12 +152,13 @@ class MemDB : public KeyValueDB
     int seek_to_first(const std::string &k);
     int seek_to_last(const std::string &k);
 
-    int seek_to_first() { return seek_to_first(NULL); };
-    int seek_to_last() { return seek_to_last(NULL); };
+    int seek_to_first() { return seek_to_first(""); };
```

Comment:
NULL is not good for cpp that is true but can we make empty string more explicit like std::string("NULL") ?